sweep/sweeper.go

```diff
@@ -944,17 +944,25 @@ func (s *UtxoSweeper) clusterByLockTime(inputs pendingInputs) ([]inputCluster,
 			cluster = make(pendingInputs)
 		}
 
-		cluster[op] = input
-		locktimes[lt] = cluster
-
-		// We also get the preferred fee rate for this input.
+		// Get the fee rate based on the fee preference. If an error is
+		// returned, we'll skip sweeping this input for this round of
+		// cluster creation and retry it when we create the clusters
+		// from the pending inputs again.
```

Comment:
In what case would an error be returned here? If an estimator for a neutrino client was down? But IIUC, that should just result in it using the cached or default value.

Reply:
Either way, indeed a bug fix as before we would add the inputs, but then the fee rate would never be set due to the continue.

Reply:
Both `bitcoind` and `btcd` estimators won't return an error from calling `feeEstimator.EstimateFeePerKW`, so the only error would be from calling `feeEstimator.EstimateFeePerKW` in `WebAPIEstimator` with a conf target of 0. Aside from that, we could get an error if the fee rate is too low(below relay fee), or the fee preference is misused, like setting both conf target and fee rate - but these should all be handled internally atm.